gazebo_ros2_control_demos/launch/cart_example_position_initial_position.launch.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) 2021 Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Comment:
Do you mind to change the copyright to open robotics ? You can add a line indicating that you are the author

Change the year too

Reply:
Please check my [comment here](https://github.com/ignitionrobotics/ign_ros2_control/pull/27#issuecomment-1017040056). I actually prefer to remove this two additional files completelly.

@bmagyar do you have any thoughts on it?

Reply:
I'm fine removing the new files

Reply:
the less content the better